webprofile/11/_index.md

```diff
@@ -23,15 +23,18 @@ The Jakarta EE Web Profile defines a profile of the Jakarta EE Platform specific
 
 * [Jakarta Web Profile 11 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/web-profile-11)
   * [Jakarta EE Platform 11 Release Plan](https://jakartaee.github.io/platform/jakartaee11/JakartaEE11ReleasePlan)
-* [Jakarta EE 11 M5 web profile Specification Document](./jakarta-webprofile-spec-11.0-M5.pdf) (PDF)
-* [Jakarta EE 11 M5 web profile Specification Document](./jakarta-webprofile-spec-11.0-M5.html) (HTML)
-* [Jakarta Web Profile 11 M5 Javadoc](./apidocs)
+* [Jakarta EE 11 web profile Specification Document](./jakarta-webprofile-spec-11.0.pdf) (PDF)
+* [Jakarta EE 11 web profile Specification Document](./jakarta-webprofile-spec-11.0.html) (HTML)
+* [Jakarta Web Profile 11 Javadoc](./apidocs)
+* [Jakarta Web Profile 11 TCK](https://www.eclipse.org/downloads/download.php?file=/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakartaeetck-11.0.0-dist.zip) ([sig TBD added by mentor](https://download.eclipse.org/jakartaee/platform/11/jakarta-jakartaeetck-11.0.0.zip.sig),[sha](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakartaeetck-11.0.0-dist-tckinfo.txt),[pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
```

Comment:
We´'re almost there. It must link to the location where the TCK files will be *after* they are published by the spec committee. They must not link to the staged ones. 

Don't worry that the link is not active yet. Just format them as if they where. No "TBD". The following can be pasted in:

* [Jakarta Web Profile 11 TCK](https://download.eclipse.org/jakartaee/platform/11/jakarta-jakartaeetck-11.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/platform/11/jakarta-jakartaeetck-11.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/platform/11/jakarta-jakartaeetck-11.0.0.zip.sha256),[pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))

Reply:
Fixed in 3740f2cba .